src/routes/transactions/handlers/proposal.rs

```diff
@@ -13,7 +11,7 @@ pub async fn submit_confirmation(
     chain_id: &str,
     safe_tx_hash: &str,
     signature: &str,
-) -> ApiResult<TransactionDetails> {
+) -> ApiResult<()> {
```

Comment:
This would be a breaking change. It is required that we keep returning the tx information